packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/SimulationResource.java

```diff
@@ -187,6 +188,19 @@ public Publisher<byte[]> stream(
 		});
 	}
 
+	@GET
+	@Path("/{jobId}/sciml-intermediate-results")
```

Comment:
Better keep matching URLs
```suggestion
	@Path("/{jobId}/sciml/partial-result")
```